tests/test_labelers.py

```diff
@@ -0,0 +1,131 @@
+import numpy as np
```

Comment:
This file should define multiple functions, one for testing each labeling function. 

We also need to test the label saving / loading logic and every method of LabeledPatients